docs/platform/longview/longview-app-for-apache-classic/index.md

```diff
@@ -0,0 +1,322 @@
+
+## Viewing Statistics
+
+To see the output for the Longview Apache App:
+
+1.  Log in to the [Linode Manager](https://manager.linode.com/).
+2.  Select the **Longview** tab.
+3.  Select the **Apache** tab.
+
+Click the image for a full-size view.
+
+[![The Longview Apache App.](1453-longview_apache_stats_sm.png)](1452-longview_apache_stats.png)
+
+You'll see the current version of Apache listed on the upper right.
+
+Mouse over a data point to see the exact numbers for that time. You can also zoom in on data points, or view older time periods with Longview Pro. For details, jump to this section in the main article about [navigating the Longview interface](/docs/platform/longview/longview/#longview-s-data-explained). The next sections cover the Longview Apache App in detail.
+
+### Requests
+
+The **Requests** graph shows the total number of requests Apache handled at the selected time. This is every HTTP and HTTPS request to your Linode.
+
+### Throughput
+
+The **Throughput** graph shows the amount of data that Apache sent and received via web requests at the time selected.
+
+### Workers
+
+The **Workers** graph shows all of the Apache workers at the selected time. The workers are broken down by state:
+
+-   Waiting
+-   Starting
+-   Reading
+-   Sending
+-   Keepalive
+-   DNS Lookup
+-   Closing
+-   Logging
+-   Finishing
+-   Cleanup
+
+### CPU
+
+The **CPU** graph shows the percentage of your Linode's CPU being used by Apache at the selected time. If you want to see the total CPU use instead, check the [Overview tab](/docs/platform/longview/longview/#overview).
+
+### Memory
+
+The **Memory** graph shows the amount of RAM being used by Apache at the selected time. If you want to see your Linode's total memory use instead, check the [Overview tab](/docs/platform/longview/longview/#overview).
+
+### Disk IO
+
+The **Disk IO** graph shows the amount of input to and output from the disk caused by Apache at the selected time. To see the total IO instead, visit the [Disks tab](/docs/platform/longview/longview/#disks).
+
+### Process Count
+
+The **Process Count** graph shows the total number of processes on your Linode spawned by Apache at the selected time. If you want to see more details, and how this stacks up against the total number of processes on your Linode, see the [Process Explorer tab](/docs/platform/longview/longview/#process-explorer).
+
+## Troubleshooting
+
+If you don't see Longview data for Apache, you'll instead get an error on the page and instructions on how to fix it. As a general tip, you can check the `/var/log/linode/longview.log` file for errors as well. You should also compare your mod\_status configuration file (typically `httpd.conf` or `status.conf`) to the example shown in Step 2 of the [Manual Configuration (All Distributions)](#manual-configuration-all-distributions) section of this article.
```

Comment:
mod_status should be in backticks

Reply:
this is classic manager guide...so disregarding